webapp/src/components/NewsList.vue

```diff
@@ -5,7 +5,7 @@
         :newsItem="newsItem"
         v-bind:key="newsItem.title"
         @updateNews="updateNews"
-        @removeNews="removeNews" 
+        @removeNews="removeNews"
```

Comment:
Ok, one thing: Event names should be kebab-cased: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names